src/screens/ConfirmationContainer/ConfirmationScreen/index.js

```diff
@@ -89,30 +102,90 @@ class ConfirmationScreen extends NavigatorComponent<Props, State> {
 
 
   buildConfirmationView() {
+    const amount = ethers.utils.bigNumberify(this.props.amount);
```

Comment:
Please extract texts from here to transactions file and i18n to get them.

Reply:
Changed